include/revng/Pipeline/Global.h

```diff
@@ -114,6 +126,16 @@ class TupleTreeGlobal : public Global {
     return llvm::Error::success();
   }
 
+  llvm::Error applyDiff(const GlobalTupleTreeDiff &Diff) override {
+    Diff.getAs<Object>()->apply(Value);
```

Comment:
No return value? Cannot fail?

Reply:
Reworked to populate `ErrorList` wit appropriate errors